cg/clients/freshdesk/freshdesk_client.py

```diff
@@ -60,3 +58,27 @@ def _configure_retries(session: Session) -> None:
         )
         adapter = HTTPAdapter(max_retries=retry_strategy)
         session.mount("https://", adapter)
+
+    @handle_client_errors
+    def reply_to_ticket(
+        self, ticket_id: str, from_email: str, reply: dict, attachments: List[Path] = None
+    ) -> None:
+        """Send a reply to an existing ticket in Freshdesk."""
+        url = f"{self.base_url}{EndPoints.TICKETS}/{ticket_id}/reply"
+
+        files = prepare_attachments(attachments) if attachments else None
+        data = {
+            "body": reply["body"],
+            "from_email": from_email,
+        }
+        response = self.session.post(url=url, data=data, files=files)
+        if response.status_code == HTTPStatus.OK:
+            LOG.info("Successfully replied to ticket %s", ticket_id)
```

Comment:
```suggestion
            LOG.info(f"Successfully replied to ticket {ticket_id}")
```

Reply:
Do you ignore `PylintW1203:logging-fstring-interpolation`: Use lazy % formatting in logging functions ?